tests_e2e/test_app.py

```diff
@@ -0,0 +1,73 @@
+import os
+from threading import Thread
+
+import pytest
+from dotenv import load_dotenv
+from selenium import webdriver
+from selenium.webdriver.common.keys import Keys
+
+import app
+from trello_client import TrelloBoard
+
+
+@pytest.fixture(scope="module")
+def driver():
+    with webdriver.Firefox() as driver:
+        yield driver
+
+
+@pytest.fixture(scope="module")
+def test_app():
+    load_dotenv(".env")
```

Comment:
This clashes with `load_dotenv(".env.test")` as only the first invoked load_dotenv() invocation is used.

This means these tests work fine when run individually but not when run altogether.

As a result you should probably:
* Combine the 2 configs
* Document this in the README

Reply:
Wouldn't it be bad practice to give the unit tests access to an actual trello board though?